server/webapp/localize_ja.xml

```diff
@@ -171,6 +171,7 @@
     <entry key="USER_SUCCESSFULLY_ADDED">User ''{0}'' successfully added.</entry>
     <entry key="USER_ALREADY_EXISTS">Failed to add user. User ''{0}'' already exists.</entry>
     <entry key="USER_DOES_NOT_EXIST_IN_DB">User ''{0}'' does not exist in the database.</entry>
+    <entry key="INVALID_LDAP_ERROR">There was an error authenticating you. Please check the server logs, or contact your the go administrator.</entry>
```

Comment:
But what if I'm the administrator? (personally, I hate error messages that say "contact the administrator" when I'm the administrator, and I have to resort to googling to figure out what to do)